app/views/users/_form.html.erb

```diff
@@ -1,17 +1,8 @@
-<%= form_for(@user) do |f| %>
-    <%= render 'shared/error_messages', object: @user %>
-
-    <%= f.label :name %>
-    <%= f.text_field :name, class: 'form-control' %>
-
-    <%= f.label :email %>
-    <%= f.email_field :email, class: 'form-control' %>
-
-    <%= f.label :password %>
-    <%= f.password_field :password, class: 'form-control' %>
-
-    <%= f.label :password_confirmation %>
-    <%= f.password_field :password_confirmation, class: 'form-control' %>
-
-    <%= f.submit yield(:button_text), class: "btn btn-primary" %>
+<%=form_for(@user) do |f| %><%=render 'shared/error_messages',
+    object: @user %><%=f.label :name %><%=f.text_field :name,
+    class: 'form-control'%><%=f.label :email %><%=f.email_field :email,
+    class: 'form-control'%><%=f.label :password %><%=f.password_field :password,
+    class: 'form-control'%><%=f.label :password_confirmation %><%=f.password_field :password_confirmation,
+    class: 'form-control'%><%=f.submit yield(:button_text),
+    class: "btn btn-primary"%>
```

Comment:
@shaniajubelag 
Did you edit this file on purpose?
Please provide proper line spacing to each element